ecommerce/frontend/src/components/SearchInput.js

```diff
@@ -14,35 +16,38 @@ const categories = [
     { key: 'womens_fashion', title: "Women's Fashion" },
 ]
 
-const { Option } = Select
+export const SearchInput = (width = 120, onSearch) => {
+    const [category, setCategory] = useState('Electronics')
```

Comment:
This won't work because there is only one prop object/argument being passed to SearchInput. Correct way is:

```js
export const SearchInput = ({width = 120, onSearch}) => {
...
}
```